core/block.js

```diff
@@ -1196,7 +1196,7 @@ Blockly.Block.prototype.jsonInit = function(json) {
   // Validate inputs.
   goog.asserts.assert(json['output'] == undefined ||
       json['previousStatement'] == undefined,
-      'Must not have both an output and a previousStatement.');
+  'Must not have both an output and a previousStatement.');
```

Comment:
This seems like the opposite of the other changes...? This is still a line continuation, right?